<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>895</width>
    <height>879</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>30</y>
     <width>591</width>
     <height>471</height>
    </rect>
   </property>
   <property name="title">
    <string>GroupBox</string>
   </property>
   <widget class="QCheckBox" name="CBReading">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>60</y>
      <width>87</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Reading</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="CBHiking">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>60</y>
      <width>87</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Hiking</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="CBSwimming">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>200</y>
      <width>87</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Swimming</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="CBDazing">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>200</y>
      <width>87</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Dazing</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="CBShopping">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>350</y>
      <width>87</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Shopping</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="CBEating">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>350</y>
      <width>87</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Eating</string>
    </property>
   </widget>
  </widget>
  <widget class="QCheckBox" name="CB3Show">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>550</y>
     <width>87</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>3 Show</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>540</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Show</string>
   </property>
  </widget>
  <widget class="QCommandLinkButton" name="CLBOpenFolder">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>650</y>
     <width>224</width>
     <height>47</height>
    </rect>
   </property>
   <property name="text">
    <string>OpenFolder</string>
   </property>
  </widget>
  <widget class="QCommandLinkButton" name="CLBOpenPage">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>740</y>
     <width>224</width>
     <height>47</height>
    </rect>
   </property>
   <property name="text">
    <string>OpenPage</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
